src/components/ImageScroller.js

```diff
@@ -88,7 +88,16 @@ export default function ImageScroller(props) {
   const classes = useStyle();
   const [maxImages, setMaxImages] = useState(MAX_IMAGES_INCREMENT);
   const imageScrollerRef = useRef(null);
-  let [rotation, setRotation] = useState(imageRotation);
+  let objRotation = {};
+  if (selectedImage) {
+    const selectedIndex = images.indexOf(selectedImage);
+    objRotation[selectedIndex] = imageRotation;
+  } else {
+    images.map((_, idx) => {
+      objRotation[idx] = imageRotation;
+    });
+  }
+  let [rotations, setRotations] = useState(objRotation);
```

Comment:
I think this is introducing unnecessary complexity.
There's no point in persisting the image rotation for other images in the frontend like this; only one value gets stored for each planter so all other rotations will be lost when they refresh (which is annoying for a user).